### ISSUE TITLE
Error on web app

### ISSUE THREAD

**Issue by diego06884** [avocode/avocode#1876]
---
* **Version**: 2.19.2-web.2
* **Web/Desktop**: Web
* **Platform**: macosx 10.12.6
* **Browser**: chrome 66.0.3344.0
* **Avocode UID**: 150849
---
```
Error: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.
    at Function.value (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:42:63589)
    at Function.value (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:42:62747)
    at t._composeLegacyLayerComposition (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:1417795)
    at t.describeLayers (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:1417374)
    at e.value (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:1646445)
    at e.value (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:1644542)
    at t._getSelectionState (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:459486)
    at t._handleSelectionChange (https://cdn.app.avocode.com/dist/v-3394921/js/avocode-web.min.js:106:458823)
    at t.<anonymous> (https://cdn.app.avoc
```

**Comment by vojtatranta** [comment]
Thank you for this report. We noticed this in our internal tracking. We will try to fix it in the 3.0.

**Comment by jakubzitny** [comment]
Closing with respect to #1567.
